cmd/notary/prettyprint_test.go

```diff
@@ -103,12 +103,12 @@ func TestPrettyPrintRootAndSigningKeys(t *testing.T) {
 	root := data.CanonicalRootRole
 
 	// add keys to the key stores
-	assert.NoError(t, keyStores[0].AddKey(keys[0].ID(), root, keys[0]))
-	assert.NoError(t, keyStores[1].AddKey(keys[0].ID(), root, keys[0]))
-	assert.NoError(t, keyStores[0].AddKey(strings.Repeat("a/", 30)+keys[0].ID(), "targets", keys[0]))
-	assert.NoError(t, keyStores[1].AddKey("short/gun/"+keys[0].ID(), "snapshot", keys[0]))
-	assert.NoError(t, keyStores[0].AddKey(keys[1].ID(), "targets/a", keys[1]))
-	assert.NoError(t, keyStores[0].AddKey(keys[2].ID(), "invalidRole", keys[2]))
+	assert.NoError(t, keyStores[0].AddKey(trustmanager.KeyInfo{Role: root, Gun: ""}, keys[0]))
+	assert.NoError(t, keyStores[1].AddKey(trustmanager.KeyInfo{Role: root, Gun: ""}, keys[0]))
+	assert.NoError(t, keyStores[0].AddKey(trustmanager.KeyInfo{Role: "targets", Gun: strings.Repeat("/a", 30)}, keys[1]))
```

Comment:
Apologies, this was my bad for just using the quoted string, but can we change these to use the constants instead?